lage.config.js

```diff
@@ -8,10 +8,10 @@ module.exports = {
     lint: ['build'],
     clean: [],
     test: ['build'],
+    'type-check': ['build'],
```

Comment:
ensures that before running `type-check` for each package, that all its dependencies have been built

Reply:
while this is a good enough hot fix we need to do better, as this will slow down whole CI. type checks are most expensive operation ATM.

This issues will be mitigated be new task orchestrator in the future, but for now can we create issue to track this ? ty

Reply:
New issue here: #25256